wolfssl/internal.h

```diff
@@ -2569,6 +2569,9 @@ struct WOLFSSL_CTX {
 #endif
     Suites*     suites;           /* make dynamic, user may not need/set */
     void*       heap;             /* for user memory overrides */
+#ifdef WOLFSSL_STATIC_MEMORY
+    byte        onHeap; /* whether the ctx/method is put on heap hint */
```

Comment:
Maybe consider using a bit field to save space. `byte onHeap:1;` and group with other bits below?

Reply:
Thanks David, updated it.